test/100-basic-name-resolution.bats

```diff
@@ -5,11 +5,61 @@
 
 load helpers
 
-@test "basic container - dns itself" {
+# custom DNS server is set to `3.3.3.3` which is invalid DNS server
+# hence all the external request must fail, this test is expected to fail
+# with exit code 124
+@test "basic container - dns itself (custom bad dns server)" {
 	setup_slirp4netns
 
 	subnet_a=$(random_subnet 5)
-	create_config "podman1" $(random_string 64) "aone" "$subnet_a" "a1" "1a"
+	create_config network_name="podman1" container_id=$(random_string 64) container_name="aone" subnet="$subnet_a" custom_dns_server="10.10.10.10" aliases='"a1", "1a"'
```

Comment:
your comment says 3.3.3.3 but here you have 10.10.10.10, both of them could become valid dns servers in the future,
I recommend to use 10.0.2.0 which is in the slirp4netns subnet and thus should not be routed elsewhere.

Reply:
fixed